.github/workflows/rust.yml

```diff
@@ -15,8 +15,12 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Build
-      run: cargo build --verbose
-    - name: Run tests
-      run: cargo test --verbose
+      - uses: actions/checkout@v2
+      - name: Ensure proper formatting (use `cargo fmt` to fix)
+        run: cargo fmt --all -- --check
```

Comment:
Please don't force formatting. Rustfmt is not good enough to apply unconditionally, and it's a bad experience for people submitting PRs to have them rejected only due to a space or something trivial that could be fixed afterwards.